uniro_admin_frontend/src/component/Map.tsx

```
@@ -0,0 +1,29 @@
+import { useEffect } from "react";
+import useMap from "../hooks/useMap";
+import useSearchBuilding from "../hooks/useUniversityRecord";
+
+type MapProps = {
+  style?: React.CSSProperties;
+};
+const Map = ({ style }: MapProps) => {
```

Biome (1.9.4)

[error] Do not shadow the global "Map" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)